01-01-intro/01-01-intro.en.sbv

```diff
@@ -1,42 +1,42 @@
 00:00 Hello, everyone. Welcome to the git screencast.
 
-This issue is an introductory video for those who didnt use git yet.
+This issue is an introductory video for those who haven't used git yet.
 
 00:12 Most likely you have already heard that Git is a Version Control System.
-To say it in other words - its a storage, a database of a project development history, that contains all the versions of its files,
-from the very old ones to current.
+In other words - its storage, a database of a project development history, that contains all the versions of its files,
+from the oldest to the most recent.
 
-00:27 We add the initial version of our files.
-And then, when we change anything in any of the added files we add the changed version to git, so git keeps track of the file's versions.
+00:27 First, we add the initial version of our files.
+And then, when we change anything in any of the added files, we add the changed version to git, so git keeps track of the file's versions.
 
-00:36 The best part of it is that we can always get a peek into it, see whats inside and restore any previous version of any of the files.
+00:36 The best part of Git is that we can always peek and see whats inside. Then we can restore the files to any previous version.
```

Comment:
Not sure about changing "it" to "Git" as in this case I referred to the fact that git keeps track of the files versions. So "it" is this feature and the best part of this feature is that we can get any version. Does it make sense?